packages/rainbowkit/src/index.ts

```diff
@@ -29,4 +29,5 @@ export { darkTheme } from './themes/darkTheme';
 export { midnightTheme } from './themes/midnightTheme';
 export { cssStringFromTheme } from './css/cssStringFromTheme';
 export { cssObjectFromTheme } from './css/cssObjectFromTheme';
+export { useRefetchBalance } from './hooks/useRefetchBalance';
```

Comment:
For these optional hooks, I think we might want to add to create a sub-package like `rainbowkit/hooks` akin to the `/wallets` package.json that we have. That way they get built separate and there is code-splitting in the bundle. I think that will be a helpful paradigm going forward as we enhance other features too